src/core/sprites/Sprite.js

```diff
@@ -490,12 +490,12 @@ Sprite.prototype._renderCanvas = function (renderer)
         {
             renderer.context.drawImage(
                 texture.baseTexture.source,
-                texture.crop.x * resolution,
-                texture.crop.y * resolution,
+                texture.crop.x * resolution * renderer.resolution,
```

Comment:
I'm also confused why this would use `renderer.resolution` for dx/dy but the drawImage just above this doesn't?